<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>371</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>VL6180X </string>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Humidité (%)</string>
      </property>
      <widget class="QLabel" name="HumidityLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Température (°C)</string>
      </property>
      <widget class="QLabel" name="temperaturelabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Pression (hPa)</string>
      </property>
      <widget class="QLabel" name="PressureLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Luminite/Distance</string>
      </property>
      <widget class="QLabel" name="DistanceBrightnessValue">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="PressureLabel_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </widget>
     <widget class="QRadioButton" name="DistanceRadio">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BrightnessRadio">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Luminosité</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>771</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string> MPU9250 </string>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Angles</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>21</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>φ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>16</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>θ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>21</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Ψ</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>120</width>
         <height>31</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="phiLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>121</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>120</width>
         <height>31</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="ThetaLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>120</width>
         <height>31</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="PsiLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="openglWidget" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>511</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>371</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Robotis dynamixel et SOMFY </string>
     </property>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Vitesse du Vent (Km/h)</string>
      </property>
      <widget class="QLabel" name="windSpeedLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="setSpeed">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="MotorSpeed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Vitesse personaliser</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Contrôle du Moteur</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
